trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -242,8 +242,7 @@ def spin(self,conditions):
             Vz  = np.repeat(Vz, Na,axis=1)
             Vz  = np.repeat(Vz, Nr,axis=2)
             
-            if rotation == None:
-                print("Propeller rotation not specified. Set to 1 (clockwise when viewed from behind).")
+            if (rotation != 1) or (rotation != -1):  
```

Comment:
This will always set the rotation to 1. Should be 'and' instead of 'or', alternatively just use 'if rotation !=-1'

Reply:
Resolved